sync_diff_inspector/source/basic.go

```diff
@@ -32,6 +32,7 @@ import (
 type BasicSource struct {
 	tableDiffs []*common.TableDiff
 	dbConn     *sql.DB
+	ctx        context.Context
```

Comment:
According to https://pkg.go.dev/context,  store `ctx` in struct seems not a good idea?